### PR TITLE
Add summary and next steps

### PR DESCRIPTION
**Pull request by markus-da**
Fixes https://github.com/digital-asset/daml-finance/issues/1074

### CODE REVIEW

File: docs/2.8.0/docs/daml-finance/tutorials/advanced-topics/instrument-modeling/date-utilities.rst [hunk 1]
```diff
@@ -283,3 +283,17 @@ the schedule end date and the payment frequency:
...
+
+Summary
+=======
+
+You have learnt about the date-related utility functions in the Daml Finance library.
+Here are some pointers regarding next steps, depending on what you want to do:
+
+- understand and validate the implementation of existing Daml Finance instruments: the
+  implementation of the Bond package could be a good starting point. It is explained and linked
+  in the :doc:`Contingent Claims Instrument <contingent-claims-instrument>` tutorial.
```
Comment (matteolimberto-da) [inline docs/2.8.0/docs/daml-finance/tutorials/advanced-topics/instrument-modeling/date-utilities.rst:295]: I think I gave a different title to that page when I reviewed the advanced topics section --> we should probably reflect that title

**Review by matteolimberto-da — COMMENTED** [review]
Minor comment

**Comment by markus-da** [comment]
@marciw-da Here is the _Conclusion / next steps_ section that you asked for in your previous review.

**Review by carrielaben-da — APPROVED** [review]
Looks good

**Review by marciw-da — APPROVED** [review]
Thanks! LGTM

**Pull request merged**